src/EasyList/TodoServiceDB.cs

```diff
@@ -46,8 +46,10 @@ public void DisplayTodo(Todo todo)
         }
         public void DisplayAllTodo(TodoOrder todoOrder)
         {
+            int Id = 1;
+
             var _todoList = _todoRepository.GetAllTodo(todoOrder)
-                            .Select(x => new { x.Id, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime() , DueDate = x.DueDate?.DateTime.ToLocalTime() });
+                            .Select(x => new { Id = Id++, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime(), DueDate = x.DueDate?.DateTime.ToLocalTime() });
```

Comment:
Here you are removing the real id and assigning a fake one, the issue is that we no longer know which this todo is inside the database since we dont have access to the id. You could do line you have done but instead of removing the real id just add a DisplayId